schedule/functional/default.yaml

```diff
@@ -0,0 +1,108 @@
+---
+name: default
+description:    >
+    Maintainer: qe-core <qe-core@suse.de>
+    Standard scenario where we mainly just follow installation
+    suggestions without any adjustments.
+conditional_schedule:
+    boot:
+        ARCH:
+            x86_64:
+                - installation/isosize
+                - '{{bootloader}}'
+            s390x:
+                - installation/bootloader_zkvm
+            aarch64:
+                - installation/isosize
+                - installation/bootloader_uefi
+            ppc64le:
+                - installation/isosize
+                - installation/bootloader
+    bootloader:
+        MACHINE:
+            64bit:
+                - installation/bootloader
+            uefi:
+                - installation/bootloader_uefi
+            64bit-smp:
+                - installation/bootloader
+            svirt-hyperv:
+                - installation/bootloader_hyperv
+                - installation/bootloader
+            svirt-hyperv-uefi:
+                - installation/bootloader_hyperv
+                - installation/bootloader_uefi
+    grub_test:
+        ARCH:
+            x86_64:
+                - installation/grub_test
+            aarch64:
+                - installation/grub_test
+            ppc64le:
+                - installation/grub_test
+    mgmt_console:
+        ARCH:
+            s390x:
+                - boot/reconnect_mgmt_console
+    system_workarounds:
+        ARCH:
+            aarch64:
+                - installation/system_workarounds
```

Comment:
Triple checking, do we apply those for  aarch64 in main.pm? I'm asking, as the intent of the default scenario is to end up with an untouched system...  I'm cool if it's already there, but it's something to review further on...

Reply:
TBH, I cannot tell.. I didn't go that much through in detail in main.pm, but mostly checking what is actually running in Functional. So, I am also in for reviewing that!